ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestionProperties.java

```diff
@@ -325,6 +340,17 @@ void validate() {
         public boolean isMappingRequired() {
             return mappingRequired;
         }
+
+        public boolean isCompressible() {
+            return compressible;
+        }
+
+        public static IngestionProperties.DataFormat getDataFormatFromString(String ingestionPropertiesDataFormat) {
```

Comment:
unused

Reply:
I intentionally left it in as it's a public method, and even though we're no longer using it, I see no reason to remove it.